mkdocs.yml

```diff
@@ -0,0 +1,119 @@
+site_name: EITprocessing
+site_description: ALIVE project
+theme:
+  name: material
```

Comment:
The `theme` key exists twice in this file, which shouldn't really happen in YML files. I am not sure which is read at the moment (this one is bare bones "material" theme, the one below has all kinds of additional details specified). Maybe test what it looks like when you remove either and stick to the one that you prefer.